src/custom/utils/operator.ts

```diff
@@ -198,20 +203,10 @@ function toApiAddress(address: string, chainId: ChainId): string {
   return address
 }
 
-export async function getFeeQuote(params: FeeQuoteParams): Promise<FeeInformation> {
-  const { sellToken, buyToken, amount, kind, chainId } = params
-  const [checkedSellAddress, checkedBuyAddress] = [checkIfEther(sellToken, chainId), checkIfEther(buyToken, chainId)]
-  console.log('[util:operator] Get fee from API', params)
-
+async function _getJson(chainId: ChainId, url: string): Promise<any> {
```

Comment:
Minor: what about making it generic, to get rid of that pesky `any`?

```suggestion
async function _getJson<T>(chainId: ChainId, url: string): Promise<T> {
```

And when calling it you might need to say the type, like `FeeInformation`, etc:

Reply:
Okay, fine with me. I thought about doing it, but passed cause:
- fetch returns `Promise<any>` and that's just a wrapper of fetch. 
- It work automatically without casting, it inferred ok the types, so I left this approach cause i don't need to add the generic information

Happy to add it too though, let me know if i miss sth

Reply:
:shrug: